lib/compiler/semantic.js

```diff
@@ -172,6 +172,7 @@ class SemanticPass {
                 /* Functions (sorted alphabetically) */
                 length: { type: 'function', exported: true, uname: 'juttle.modules.string.length', arg_count: 1             },
                 concat: { type: 'function', exported: true, uname: 'juttle.modules.string.concat', arg_count: [0, Infinity] },
+                fromCharCode: { type: 'function', exported: true, uname: 'juttle.modules.string.fromCharCode', arg_count: [1, Infinity]},
```

Comment:
What? `arg_count` is the number of arguments, and `String.fromCharCode()` doesn't make sense.